docs/USAGE.md

```diff
@@ -67,40 +67,58 @@ There are two ways to create an `LVMVolumeGroup` resource:
 
     ```yaml
     apiVersion: storage.deckhouse.io/v1alpha1
-    kind: LvmVolumeGroup
+    kind: LVMVolumeGroup
     metadata:
       name: "vg-0-on-node-0"
     spec:
       type: Local
-      blockDeviceNames:
-        - dev-c1de9f9b534bf5c0b44e8b1cd39da80d5cda7c3f
-        - dev-f3269d92a99e1f668255a47d5d3500add1462711
-      actualVGNameOnTheNode: "vg-0"
+      local:
+        nodeName: "node-0"
+      blockDeviceSelector:
+        matchExpressions:
+        - key: kubernetes.io/metadata.name
+          operator: In
+          values:
+          - dev-07ad52cef2348996b72db262011f1b5f896bb68f
+          - dev-e90e8915902bd6c371e59f89254c0fd644126da7
+    matchLabels:
+      kubernetes.io/hostname: node-0
```

Comment:
```suggestion
```

Reply:
I think it would be confusing for users. `matchLabels` doesn't make any sense in this case. We should add a separate example with `matchLabels` without the kubernetes.io/metadata.name expression.